Makefile

```diff
@@ -20,6 +20,9 @@ dev: test
 # Verifications to run before sending a pull request
 pr: black-check init dev
 
+# Verifications to run before sending a pull request, skipping black check because black requires Python 3.6+
+pr2.7: init dev
```

Comment:
Nit: assuming existing code is `black`-compliant, would be nice if we didn't have to introduce new Python 2-specific cruft, but I'm assuming it doesn't play too well with `pip` et al. 🤒

Reply:
When we run CI, there are multiple env including Python2.7, 3.6, 3.7 and 3.8. The code style will be checked in Python3 .X though. Here we just skip the check in one of the env, I think it should be ok 🤔

Reply:
FYI, why the old way (download black binary directly) won't work:

https://github.com/psf/black/issues/1669